recipes/libsafec/all/conanfile.py

```diff
@@ -87,7 +83,8 @@ def package(self):
             tools.remove_files_by_mask(".", "*.la")
 
     def package_info(self):
-        self.cpp_info.libs = ["safec"]
+        self.cpp_info.includedirs = ["include/libsafec"]
```

Comment:
There's no reason to force consumers to by-pass installed folder layout

```suggestion
```

Reply:
I agree, but that's what the original package does. I would rather have only `include`, but that would break compatibility?
https://github.com/rurban/safeclib/blob/0234bec46da4863f849f100c2f5336412ab2f69b/libsafec.pc.in#L11
I checked Debian packages, they use the same file.

Reply:
Let's meet in between?
```suggestion
        self.cpp_info.includedirs.append([os.path.join("include", "libsafec")])
```

Reply:
@madebr I have checked newer glibc for these functions. I guess I would insist on keeping _only_ `include/libsafec` path. Else we introduce a possibility of future incompatibility for user code.

~~~
safe_mem_lib.h(3)                              safec                              safe_mem_lib.h(3)

NAME
       safe_mem_lib.h

SYNOPSIS
       #include 'safe_config.h'
       #include 'safe_lib_errno.h'
       #include 'safe_types.h'
       #include 'safe_compile.h'
       #include <wchar.h>
~~~

Reply:
In cases like these, I try to make the difference between using a conan package and one from your linux distribution as small as possible.
A linux development package of libsafec would install it in `/usr/include/libsafec`.
This means that they can be used as e.g. `#include <libsafec/safe_lib_errno.h?`.

Reply:
That's a good observation! 
I also checked the standard, it has nothing about special headers. So safe_mem_lib.h could be something very specific to my OS.
https://en.cppreference.com/w/c/string/byte/memcpy

I guess we would keep the the includepath to mimic /usr/include then.